pyproject.toml

```
@@ -0,0 +1,62 @@
+# Prepare a release:
+#
+#  - git pull --rebase
+#  - update version in pyperf/__init__.py and doc/conf.py
+#  - set release date in doc/changelog.rst
+#  - git commit -a -m "prepare release x.y"
+#  - Remove untracked files/dirs: git clean -fdx
+#  - run tests: tox --parallel auto
+#  - git push or send the PR to the repository
+#  - check Github Action CI: https://github.com/psf/pyperf/actions/workflows/build.yml
+#
+# Release a new version:
+#
+#  - go to the GitHub release tab: https://github.com/psf/pyperf/releases
+#  - click "Draft a new release" and fill the contents
+#  - finally click the "Publish release" button! Done!
+#  - monitor the publish status: https://github.com/psf/pyperf/actions/workflows/publish.yml
+#
+# After the release:
+#
+#  - set version to n+1
+#  - git commit -a -m "post-release"
+#  - git push or send the PR to the repository
+
+[build-system]
+requires = ["setuptools >= 35.0.2", "setuptools_scm >= 2.0.0, <3", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyperf"
+dynamic = ["version"]
+license = {text = "MIT license"}
```

```suggestion
license = {text = "MIT"}
```

PEP 639 is still in work, but in the meantime its worth still using standard SPDX identifiers here, and repeating "license" is rather redundant given this is the "license" field, after all